src/web/docusaurus/static/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Telescope Documentation",
+  "short_name": "Telescope Documentation",
+  "theme_color": "#121D59",
+  "background_color": "#FFFFFF",
```

Comment:
Do we want to make use of the theme colours defined here
https://github.com/Seneca-CDOT/telescope/blob/232e0a98a852b16335bf610a9e0620311f95ab96/src/web/docusaurus/src/css/custom.css#L10
https://github.com/Seneca-CDOT/telescope/blob/232e0a98a852b16335bf610a9e0620311f95ab96/src/web/docusaurus/src/css/custom.css#L28

Reply:
I think we can address this in another issue. I've tried setting `"theme_color": "var(--ifm-color-primary)"` but Chrome dev tools said it was invalid and automatically set the colour to white and I don't want to make this PR bigger than it already is.

Reply:
I think it might be
```js
"theme_color": var(--ifm-color-primary)
```
without the quotes around `var()`
but, I am quite weak at front-end so I'll let the front-end gurus decide what to do

Reply:
That's what I initially thought, too, but apparently 
```
"theme_color": var(--ifm-color-primary)
``` 
is not valid JSON... this can totally be someone else's issue ;D

Reply:
I don't think you can use `var` see https://web.dev/learn/pwa/web-app-manifest/#recommended-fields